src/inference/src/compilation_context.cpp

```diff
@@ -121,8 +122,32 @@ std::string ModelCache::compute_hash(const std::string& modelStr,
 
         auto ptr = static_cast<size_t*>(tensor.data());
         size_t size = tensor.get_size() / sizeof(size_t);
-        for (size_t i = 0; i < size; i++)
-            seed = hash_combine(seed, ptr[i]);
+
+        size_t block_size = 100000000;
+        size_t blocks_num = size / block_size;
+        std::vector<uint64_t> block_hashes(blocks_num + 1);
```

Comment:
is it initialized with zeros by default?

Reply:
fixed